scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/TaskData.java

```diff
@@ -46,6 +46,7 @@
 import javax.persistence.OneToMany;
 import javax.persistence.OneToOne;
 import javax.persistence.OrderColumn;
+import javax.persistence.QueryHint;
```

Comment:
I don't see where  javax.persistence.**QueryHint** is used

Reply:
I was using it, but I removed it because it was creating issues.
I am going to remove the imports